frontend/components/SentTransfers.vue

```diff
@@ -55,17 +55,6 @@
               </nuxt-link>
             </p>
           </template>
-          <template #cell(hash)="data">
-            <p class="mb-0">
-              <nuxt-link
-                v-b-tooltip.hover
-                :to="`/block?blockNumber=${data.item.block_number}`"
-                title="Check block information"
-              >
-                {{ shortHash(data.item.hash) }}
-              </nuxt-link>
-            </p>
-          </template>
```

Comment:
Why do we need to remove this layout?

Reply:
Events do not have a hash, but initially extrinsics do have a hash. Transfers are a part of events. You can find more information regarding this change in the following link: https://github.com/Cerebellum-Network/cerestats/pull/150#issue-1789121443. I explained why I replaced extrinsics with events.